en/advanced_config/esc_calibration.md

```diff
@@ -48,6 +48,6 @@ To calibrate the ESCs:
 
 :::note
 High-quality controllers come with a factory calibration. 
-In *theory* this means that they can be configured by just setting the [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) and [PWM_MAX](../advanced_config/parameter_reference.md#PWM_MAX) parameters to the values provided in the ESC technical specification.
+In *theory* this means that they can be configured by just setting the [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) and [PWM_MAX](../advanced_config/parameter_reference.md#PWM_MAX) parameters to the values provided in the ESC technical specification. 

```

Comment:
@bkueng PWM_MIN and PWM_MIN are no longer in the parameters. Do you know what I can replace them with here?

Reply:
PWM_MAIN_MAX/PWM_AUX_MAX